lib/colorls/core.rb

```diff
@@ -306,8 +301,8 @@ def options(path, content)
     def tree_traverse(path, prespace, indent)
       contents = init_contents(path)
       contents.each do |content|
-        icon = content == contents.last || Dir.exist?("#{path}/#{content}") ? ' └──' : ' ├──'
-        print tree_branch_preprint(prespace, indent, icon).colorize(@colors[:tree])
+        content == contents.last || Dir.exist?("#{path}/#{content}") ? ' └──' : ' ├──'
+        print tree_branch_preprint(prespace, indent, content).colorize(@colors[:tree])
```

Comment:
Breaks `colorls -t`. Please revert these 2 lines.

Reply:
I'll try to make some tests for it... :)